contracts/Strategy.sol

```diff
@@ -76,18 +79,21 @@ contract Strategy is BaseStrategyInitializable, ICallee {
     uint256 public minRatio = 0.005 ether;
     uint256 public minRewardToSell = 1e15;
 
+    enum SwapRouter {UniV2, SushiV2, UniV3}
+    SwapRouter public swapRouter = SwapRouter.UniV2; // only applied to aave => want, stkAave => aave always uses v3
+
     bool public sellStkAave = true;
     bool public cooldownStkAave = false;
-    bool public useUniV3 = false; // only applied to aave => want, stkAave => aave always uses v3
     uint256 public maxStkAavePriceImpactBps = 1000;
 
     uint24 public stkAaveToAaveSwapFee = 10000;
     uint24 public aaveToWethSwapFee = 3000;
     uint24 public wethToWantSwapFee = 3000;
 
-    uint16 private constant referral = 0; // Aave's referral code
     bool private alreadyAdjusted = false; // Signal whether a position adjust was done in prepareReturn
 
+    uint16 private constant referral = 0; // Aave's referral code
```

Comment:
change this to 7 (Yearn's referral code)

it doesn't work now but i've been told by Aave team that this could be reactivated retroactively and it's free to send 7 instead of 0 just in case